core/src/main/scala/org/bitcoins/core/psbt/PSBTRecord.scala

```diff
@@ -162,6 +167,21 @@ object InputPSBTRecord extends Factory[InputPSBTRecord] {
     override val value: ByteVector = signature.bytes
   }
 
+  object PartialSignature extends Factory[PartialSignature] {
+
+    def dummyPartialSig: PartialSignature =
```

Comment:
nit: might as well make this a `val` no?

Reply:
I think `def` makes more sense here that way you get a different one every time. This is useful because you can't have a duplicate key in a `PSBTMap`

Reply:
In that case I would prefer
```
def dummyPartialSig(pubKey: ECPublicKey = ECPublicKey.freshPublicKey): PartialSignature = {
    PartialSignature(pubKey, DummyECDigitalSignature)
}
```